docs/source/dev_guide.rst

```diff
@@ -54,16 +54,21 @@ the procedure. If not, here is how you do this:
 * `Youtube - How To Pull Request in 3 Minutes <https://www.youtube.com/watch?v=jRLGobWwA3Y>`_
 
 Documentation
-.............
+-------------
 
 The `PiFinder documentation <https://pifinder.readthedocs.io/en/release/index.html>`_
 is written in `reStructuredText <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links>`_ . 
-The files are located in PiFinders GitHub repository under ``docs`` and have 
-the ending ``.rst``. Many open source projects use `redthedocs.io <eadthedocs.io>` 
-for creating documentation since it is immediately generated, when you are committing 
-the GitHub code (CI/CD pipeline). It is very easy to link your fork of the documentation 
-code to GitHub. 
+The files are located in PiFinders GitHub repository under ``docs/source`` and have 
+the ending ``.rst``. We publish the documentation to `redthedocs.io <eadthedocs.io>`_ manually. 
```

Comment:
Thanks for adding the section on previewing docs locally, it's very useful!   The information about how the documentation is generated is probably a bit unclear, but it is automatic.

Let's revert the change above here and I'll clarify a little bit about how this works as my comment about the CI/CD pipeline and fork linkage is pretty misleading :+1:

Reply:
Done. I rewrote the section already, see the new commit.